src/components/Lend/LoanSearch/LoanSearchSectorFilter.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<form class="tw-flex tw-flex-col tw-gap-1.5 tw-mb-2" @submit.prevent>
+		<fieldset>
+			<kv-checkbox-list :items="items" @updated="updateSectors" />
+		</fieldset>
+	</form>
+</template>
+
+<script>
+import KvCheckboxList from '@/components/Kv/KvCheckboxList';
+import { getUpdatedSectors } from '@/util/loanSearchUtils';
+
+export default {
+	name: 'LoanSearchSectorFilter',
+	components: {
+		KvCheckboxList,
+	},
+	props: {
+		/**
+		 * The sectors list to display has checkboxes. Expected format:
+		 *   sectors: [
+		 *     {
+		 *       id: 1,
+		 *       name: '',
+		 *     }
+		 *   ],
+		 */
+		sectors: {
+			type: Array,
+			default: () => []
+		}
+	},
+	data() {
+		return {
+			displayedSectors: this.sectors,
+			filterResults: []
+		};
+	},
+	computed: {
+		items() {
+			return this.displayedSectors.map(c => ({
+				value: c.id.toString(),
+				title: c.name,
+			}));
+		}
+	},
+	methods: {
+		updateSectors(sectors) {
+			this.filterResults = sectors.map(sector => parseInt(sector, 10));
```

Comment:
Seeing as this isn't used outside of this scope, it could probably just be a local const. Not a blocker to merge.